plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -108,6 +111,10 @@ export class ElasticSearchSearchEngine implements SearchEngine {
           username: authConfig.getString('username'),
           password: authConfig.getString('password'),
         },
+        ssl: {
```

Comment:
I'm guessing this is safe to add because `true` is the default value in the underlying client?  But I'd be a little more comfortable if, when no config is provided, the `ssl` key isn't even passed to the client at all.  Maybe something like...

```typescript
return new Client({
  ...(sslConfig ? { ssl: { rejectUnauthorized: value } } : {}),
});
```

Reply:
That was the idea, but I prefer your suggestion. Thanks.